### DISCUSSION TITLE
Has this project stagnated?

### DISCUSSION THREAD

**Discussion by mwoodpatrick** [k3d-io/vscode-k3d#25] [General]
It looks like this project has stagnated is there a reason for this? Seems true also for the KinD vscode plugin.

It would be helpful to have tools for managing clusters & other Kubernetes resources integrated with vscode, intellisense & github copilot for yaml files etc.
	
What vscode plugins are folks using in their k3d/Kubernetes development?

**Comment by iwilltry42** [comment]
I guess that we just need more collaborators/maintainers over here :+1: 

Th3 k3d config file schema is published as a jsonschema, so you don't need a plugin for that part.
Though I'd actually like to have it in the plugin as well..
